diff --git a/node_modules/@astrojs/vercel/dist/serverless/adapter.js b/node_modules/@astrojs/vercel/dist/serverless/adapter.js
index ae8cd14..9bc03ef 100644
--- a/node_modules/@astrojs/vercel/dist/serverless/adapter.js
+++ b/node_modules/@astrojs/vercel/dist/serverless/adapter.js
@@ -36,6 +36,8 @@ const ISR_PATH = `/_isr?${ASTRO_PATH_PARAM}=$0`;
 const SUPPORTED_NODE_VERSIONS = {
     18: { status: 'retiring', removal: 'Early 2025', warnDate: new Date('October 1 2024') },
     20: { status: 'default' },
+    22: { status: 'default' },
+    24: { status: 'default' },
 };
 function getAdapter({ edgeMiddleware, functionPerRoute, middlewareSecret, skewProtection, }) {
     return {
